src/auth/RegisterPage.tsx

```diff
@@ -54,8 +54,9 @@ export const RegisterPage: FC = () => {
   const [error, setError] = useState<Error>();
   const [password, setPassword] = useState("");
   const [passwordConfirmation, setPasswordConfirmation] = useState("");
-  const [privacyPolicyUrl, recaptchaKey, register] =
-    useInteractiveRegistration();
+  const interactiveRegistration = useInteractiveRegistration();
```

Comment:
This is not realted to your pr but i dont like how: `useInteractiveRegistration` is returning an array and `useRegisterPasswordlessUser` returns an object.